### PR TITLE
Define a Circle's Radius with Physical Units

### LINKED ISSUE #1032: Define a Circle based on Physical Units
The Android Maps team just posted a proof of concept for using `TurfTransformation#circle()` to 
create a `FillLayer` that defines a Circle in physical units (i.e. "meters") rather than in pixels. The original PR is here: https://github.com/mapbox/mapbox-gl-native/pull/14429

This would be a great addition to the demo app.

cc/ @langsmith @osana

### PR DESCRIPTION
**Pull request by riastrad**
Resolves #1032 

This PR demonstrates how to add a circle to the map with a radius specified in various physical units. It leverages `TurfTransformation#circle()` so it is being added under the "Java Services" section of the app.

cc: @langsmith

### CODE REVIEW

**Comment by langsmith** [comment]
@riastrad , you see yourself being able to work on this in the near(ish) future? Or should I take over on this pr?

**Comment by riastrad** [comment]
@langsmith I don't foresee being able to pick this up within the next couple of weeks 😬

**Comment by langsmith** [comment]
Ok, no problem

**Comment by langsmith** [comment]
Ok, this is ready for some review. 

ezgif com-resize (9)

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/TurfPhysicalCircleActivity.java [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+    }
+    drawPolygonCircle(lastClickPoint);
+  }
+
+  @Override
+  public void onNothingSelected(AdapterView<?> adapterView) {
+    // Empty on purpose. Not used in this example.
+  }
+
+  private void adjustTextView(int string, TextView textView, int progress, int difference) {
```
Comment (LukasPaczos) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/TurfPhysicalCircleActivity.java]: `difference` seems unnecessary here

**Pull request merged**